dd-sdk-android-gradle-plugin/src/test/kotlin/com/datadog/gradle/plugin/internal/OkHttpUploaderTest.kt

```diff
@@ -275,6 +279,111 @@ internal class OkHttpUploaderTest {
             )
     }
 
+    @Test
+    fun `M throw a MaxSizeExceededException W upload() { mappingFile size exceeded 50 MB}`(
+        forge: Forge
+    ) {
+        // GIVEN
+        val fakeFileSize =
+            forge.aLong(min = OkHttpUploader.MAX_MAP_FILE_SIZE_IN_BYTES + 1)
+        val fakeTooLargeMappingFile = spy(File(tempDir, forge.anAlphabeticalString()))
+        doReturn(fakeFileSize)
+            .whenever(fakeTooLargeMappingFile).length()
+
+        // THEN
+        assertThrows<MaxSizeExceededException>(
+            {
+                testedUploader.upload(
+                    mockWebServer.url("/").toString(),
+                    fakeTooLargeMappingFile,
+                    fakeRepositoryFile,
+                    fakeIdentifier,
+                    fakeRepositoryInfo
+                )
+            }
+        )
+    }
+
+    @Test
+    fun `M fail and do nothing W upload() { mappingFile size exceeded 50 MB }`(
```

Comment:
seems like this test is equal to the test above and the difference is just additional `assertThat(mockWebServer.requestCount).isEqualTo(0)` assertion. maybe this test can be removed and assertion merged with test above?

Reply:
yes...we do this intentionally as we want to have 1 test/assertion. This way if your test fails in CI you know exactly what it  failed (either sending a log message, either skipping the writer call)

Reply:
I do agree with Nikita, you're duplicating the assertThrows, and simply adding one assertion. Either make it different (only assert on the `mockWebServer.requestCount` here and on the exception above), or combine the two